person.rb

```diff
@@ -1,14 +1,15 @@
 require './nameable'
 class Person < Nameable
```

Comment:
- [ ] You'll equally need a method for adding rental in this class. [ReadMore..](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)